### PR TITLE
Feature/front end testing

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Ref https://github.com/DemocracyClub/dc_django_utils/issues/6

### CODE REVIEW

**Comment by michaeljcollinsuk** [comment]
Looks good so far! I think what will be useful for you to do is install your local dc utils branch with this change in to another projects env (I think you use the `pip install -e path/to/project` to do that). Or will need to merge this and create a new release. So that you can then try running some tests in the projects that are going to use the helper

**Comment by michaeljcollinsuk** [comment]
Have updated to a point where I think this is ready to be merged and made in to a release, so that the `validate_html` method can be used in other projects. Other changes:
- Tests can now be run in local development with a simple `pytest`
- In order to get this working, as the project and some of the helpers are is dependent on Django, I have added a `tests` project to help run tests. Open to feedback about this is the best course of action - I found resources hard to come by, ended up taking a look at other similar utils projects such as `django-extensions` as well as our own `dc_signup_form` repo.
- Started to use precommit for black/djhtml
- Refactored the requirements files, to help with next work to add tox and circleci

File: setup.cfg [hunk 1]
```diff
@@ -13,8 +13,11 @@ install_requires =
     django-pipeline == 2.0.6
     libsass == 0.21.0
     jsmin==2.2.2
-    django-pytest
+    django<3
+    pytest-django
+    pytest
     pytidylib
+    dc_signup_form @ git+https://github.com/DemocracyClub/dc_signup_form.git@2.1.0
```
Comment (symroe) [inline setup.cfg:20]: Why is this a requirement?

File: tests/testapp/settings.py [hunk 2]
```diff
@@ -31,13 +31,13 @@
...
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    "dc_utils",
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'dc_utils',
```
Comment (symroe) [inline tests/testapp/settings.py:40]: Can you explain this quotation mark change? I think `black` will just change it back?

Comment (michaeljcollinsuk) [reply tests/testapp/settings.py:40]: Looks like I missed running black on the tests app - will update

**Pull request merged**